simple-wp-helpdesk/frontend/class-portal.php

```
@@ -484,26 +520,35 @@ function swh_render_lookup_form() {
 						$new_token = wp_generate_password( 20, false );
 						update_post_meta( $lt->ID, '_ticket_token', $new_token );
 						update_post_meta( $lt->ID, '_ticket_token_created', time() );
 						$link = swh_get_secure_ticket_link( $lt->ID );
 						if ( $link ) {
-							$uid           = get_post_meta( $lt->ID, '_ticket_uid', true );
+							$uid           = swh_get_string_meta( $lt->ID, '_ticket_uid' );
 							$ticket_links .= '- ' . $uid . ': ' . $lt->post_title . "\n  " . $link . "\n\n";
+						} else {
+							// phpcs:ignore WordPress.PHP.DevelopmentFunctions.error_log_error_log -- Intentional; logs link generation failures for admin troubleshooting.
+							error_log( 'SWH: swh_get_secure_ticket_link() returned false for ticket ' . $lt->ID . ' during lookup for ' . $lookup_email );
 						}
```

PHPMD (2.15.0)

[error] The method swh_render_lookup_form uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (undefined)

(ElseExpression)